### ISSUE TITLE
Compilation problem with Cuda 10.2

### ISSUE THREAD

**Issue by MCvin** [nicolas-chaulet/torch-points3d#349]
Hi,

I am trying to install torch-points3d with pip and I get the following error 6 times:
```
    cuda/src/chamfer_dist.cu(204): error: no instance of overloaded function "atomicAdd" matches the argument list
                argument types are: (double *, double)
              detected during instantiation of "void chamfer_dist_grad_kernel(int, int, const scalar_t *, int, const scalar_t *, const scalar_t *, const int *, scalar_t *, scalar_t *) [with scalar_t=double]"
    (225): here
```
This happens with Cuda 10.2 and 11.0.
I compiled torch 1.5.1 from source.

Maybe I'm missing something but I can't find out.

**Comment by nicolas-chaulet** [comment]
Hi,
Sorry to hear that... This is more an issue for torch points kernels I believe:
https://github.com/nicolas-chaulet/torch-points-kernels

I just compiled it without troubles (cuda 10.2). You can also use the version before last (0.6.6) and this should go away:
```
pip install torch-points-kernels==0.6.6
```
then install torch-points3d

**Comment by MCvin** [comment]
Thank you !

Sorry yes it is more about torch points kernels. I installed the previous version 0.6.6 as you suggested 
and then I could successfully install torch-points3d.

**Comment by nicolas-chaulet** [comment]
@MCvin this looks like you ahve an older GPU that does not support AtomicAdd, could you please confirm?

**Comment by MCvin** [comment]
@nicolas-chaulet you are right, it seems it is the problem. From the CUDA documentation (https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html):
"The 64-bit floating-point version of atomicAdd() is only supported by devices of compute capability 6.x and higher."
And the card I am using is a GeForce GTX TITAN Black with a compute capability of 3.5...

**Comment by nicolas-chaulet** [comment]
I am closing this for now, if you really need the versions of the kernels, please open a new issue there.

**Comment by nicolas-chaulet** [comment]
Actually @CCInc solved that:
https://github.com/nicolas-chaulet/torch-points-kernels/pull/52

It should be available in version 0.6.8

**Comment by Tian-Yifei** [comment]
my cuda version 11.0 and i can install torch-points-kernels by

pip install torch-points-kernels==0.6.6
